.github/actions/analytics-uploader-wrapper/action.yaml

```diff
@@ -63,6 +66,19 @@ runs:
       env:
         TRUNK_PUBLIC_API_ADDRESS: https://api.dev.trunk-staging.io/
 
+    - name: Upload test results with CLI version ${{ inputs.cli-version }} to dev2
+      if: "${{ inputs.cli-version != '' }}"
+      uses: trunk-io/analytics-uploader@main
```

Comment:
Please add continue-on-error to true to the previous task.
Basically, last one doesn't have it, but since you are adding a new one, you need to add it to the previous